### DISCUSSION TITLE
How to integrate with input color

### DISCUSSION THREAD

**Discussion by GabrielGomesBarbosa** [react-hook-form/react-hook-form#2405] [Q&A] [answered]
Hi @bluebill1049, im trying to use `react-hook-form` with `input type="color"` but i cant, i tried many ways but screen stay too slow, or the color dont be fixed on hit save, using `<Controller>` but i always get some error.

Can you show me a example?

Im using v5. Thanks

**Comment by bluebill1049** [comment, marked as answer]
can you share a codesandbox?

**Reply by GabrielGomesBarbosa** [reply]
Of course... @bluebill1049 i make it work, but its there is a new way correctly to do that ?

https://codesandbox.io/s/exciting-chatterjee-kfo7g?file=/src/App.js

Thanks.

**Reply by bluebill1049** [reply]
looks good, don't use `getValues` like that, this is getting fixed on the minor release. 

`getValues` will not return defaultValue in the next update

**Comment by onyedikachi23** [comment]
Using `<Controller>` works best for this type of input, especially if you want to listen to changes on it.
